DFC.ServiceTaxonomy.CustomEditor/Configuration/PagesConfiguration.cs

```diff
@@ -4,5 +4,7 @@ public class PagesConfiguration
     {
         public string? PublishedAppPrefix { get; set; }
         public string? PreviewAppPrefix { get; set; }
+        public string? PublishedJobProfileAppPrefix { get; set; }
```

Comment:
Could move them to their own class JobProfileConfiguration. both these can inherit from the same base class as well.

Reply:
Something like BasePreviewConfiguration